pkg/transparentproxy/transparentproxy_experimental.go

```diff
@@ -54,7 +53,7 @@ func ShouldEnableIPv6(port uint16) (bool, error) {
 	// kernel module which is adding `nat` table, so we are checking if this table
 	// exists and if so, we are assuming we can safely proceed with ip6tables
 	// ref. https://github.com/kumahq/kuma/issues/2046
-	err = exec.Command(constants.IP6TABLES, "-t", constants.NAT, "-L").Run()
+	err = exec.Command("ip6tables", "-t", "nat", "-L").Run()
```

Comment:
why can't we use constatns here?

Reply:
3 reasons:

1. These constants come from the istio package we're going to remove
2. having a constant that is exactly equal to the value of the constant is questionnable in usefulness
3. constants are not easily lintable and look too much like variables and it makes gosec point out a possible issue with inject.

Reply:
did not notice that it came from istio package, 👍